src/pilot/dracclient_raid.patch

```diff
@@ -0,0 +1,54 @@
+--- raid.py	2021-09-16 14:39:49.035032177 -0500
++++ raid.new	2021-09-16 14:38:50.658107416 -0500
+@@ -911,6 +911,15 @@
+         RAID to JBOD or vice versa.  It does this by only converting the
+         disks that are not already in the correct state.
+ 
++        When converting H755 RAID controller physical disks to non-RAID mode,
++        RAID-0 virtual disks get created for each physical disk and disks moved
++        to 'Online' state.
++
++        This is different from other controllers supporting non-RAID conversion
++        and takes up physical disks that cannot be used for user intended RAID
++        configuration later. H755 RAID controllers are excluded from converting
++        to non-RAID mode leaving disks in 'Ready' state.
```

Comment:
This comment block has been modified upstream during the code review process. Please change this to the following:

    + +        When converting Dell EMC PERC H755 RAID controller physical disks
    + +        to non-RAID mode, RAID-0 virtual disks get created for each physical
    + +        disk and disks moved to 'Online' state.
    + +
    + +        This is different from other controllers supporting non-RAID conversion
    + +        and takes up physical disks that cannot be later used for user intended
    + +        RAID configuration. H755 RAID controllers are excluded when converting
    + +        to non-RAID mode leaving disks in 'Ready' state.